quetz/testing/fixtures.py

```diff
@@ -144,6 +144,8 @@ def config_base(database_url, plugins):
 
 [plugins]
 enabled = {plugins}
+
+[redis]
```

Comment:
make sure that the tests also pass if [redis] section is not present, you can check that in the code using `config.configured_section("redis")`

Reply:
can you also add the extra config settings to the sphinx docs, it can be done in a separate PR

Reply:
yeah, thanks, I didn't know that we wanted to make it pass even without the `[redis]` section. Will do the docs too.